README.md

```diff
@@ -30,6 +30,11 @@ repos:
 This hook matches any paths ending in `packer.json` and `.pkr.hcl` by default.
 File paths can be added for checking manually as additional arguments.
 
+## Notes about the `packer_fmt` hook ##
+
+This hook will reformat any Packer HCL files that fail their `packer fmt -check`
+test.
+
```

Comment:
Rather than add a blurb here I would prefer the description for the hook at

https://github.com/cisagov/pre-commit-packer/blob/9871013c54eaae9c38bfed6c512c28f30f105c66/README.md#L13

be updated to reflect the change in functionality.

Reply:
Will do - makes much more sense there.